spec/phase2.html

```diff
@@ -297,6 +320,766 @@
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>ITransformProvider::Rotate</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        <div>IWindowProvider::SetVisualState (minimized, maximized, normal)</div>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>ACTION_ACCESSIBILITY_FOCUS</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        <div>ACTION_CLEAR_ACCESSIBILITY_FOCUS</div>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        <div>ACTION_NEXT_AT_MOVEMENT_GRANULARITY</div>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>ACTION_NEXT_HTML_ELEMENT</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        <div>ACTION_PREVIOUS_AT_MOVEMENT_GRANULARITY</div>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>ACTION_PREVIOUS_HTML_ELEMENT</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>atk_editable_text_set_run_attributes</td>
+        </tr>
+        <tr>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>atk_editable_text_set_text_contents</td>
```

Comment:
Agreed, I could pull them out of the table and just list things that aren't very universal